src/dmd/semantic3.d

```diff
@@ -875,7 +875,6 @@ extern(C++) final class Semantic3Visitor : Visitor
                 sc2.flags = (sc2.flags & ~SCOPEcontract) | SCOPErequire;
 
                 // BUG: need to error if accessing out parameters
```

Comment:
Isn't this already forbidden?

https://run.dlang.io/is/NacqYo

Reply:
But is that accessing the `out` parameter, or `out` parameters?

Reply:
Actually, I've worked out what this means:
```
static int cache;

int foo(out int bar)
in
{
    cache = bar;  // This should be an error.
}
```

Reply:
https://issues.dlang.org/show_bug.cgi?id=18203